fdbcli/fdbcli.actor.cpp

```diff
@@ -1698,7 +1698,7 @@ ACTOR Future<int> cli(CLIOptions opt, LineNoise* plinenoise) {
 
 				if (tokencmp(tokens[0], "profile")) {
 					getTransaction(db, managementTenant, tr, options, intrans);
-					bool _result = wait(makeInterruptable(profileCommandActor(tr, tokens, intrans)));
+					bool _result = wait(makeInterruptable(profileCommandActor(localDb, tr, tokens, intrans)));
```

Comment:
I think this needs to use the `Reference<IDatabase> db` rather than `localDb`. Otherwise, this command wouldn't be usable across client versions.

Edit: we can defer this change until later once the profile command is updated to use special keys.

Reply:
Created follow up task over here https://github.com/apple/foundationdb/issues/7091